tools/test_configs/HeapBlockDeviceAndWifiInterface.json

```diff
@@ -1,5 +1,41 @@
 {
     "config": {
+        "wifi-secure-ssid": {
+            "help": "WiFi SSID for WPA2 secured network",
+            "value": "\"WIFI_SSID_SECURED\""
+        },
+        "wifi-unsecure-ssid": {
+            "help": "WiFi SSID for unsecure netwrok",
+            "value": "\"WIFI_SSID_UNSECURE\""
+        },
+        "wifi-password": {
+            "help": "WiFi Password",
+            "value": "\"WIFI_PASSWORD\""
+        },
+        "wifi-secure-protocol": {
+            "help": "WiFi security protocol, valid values are WEP, WPA, WPA2, WPA_WPA2",
+            "value": "\"WIFI_SECURITY\""
+        },
+        "wifi-ch-secure": {
+            "help": "Channel number of secure SSID",
+            "value": WIFI_SECURE_CH
```

Comment:
This is a bit dangerous..

The JSON file should be buildable without modifications.
So preferably use some known values that can be search&replaced in CI jobs, instead of markers that break the normal command line build.